server/routerlicious/packages/services-shared/src/runner.ts

```diff
@@ -63,11 +65,14 @@ export function runService<T extends IResources>(
     runningP.then(
         () => {
             logger?.info("Exiting");
+            Lumberjack.info("Exiting");
             process.exit(0);
         },
         (error) => {
             logger?.error(`${group} service exiting due to error`);
+            Lumberjack.error(`${group} service exiting due to error`);
```

Comment:
Otherwise, if you prefer to keep the Lumberjack logs instead, maybe we can combine line 73 and 75 in a single line like:
```typescript
Lumberjack.error(`${group} service exiting due to error`, undefined, error);
```